trame_slicer/segmentation/segmentation.py

```diff
@@ -522,6 +522,26 @@ def set_segment_labelmap(self, segment_id, label_map: vtkImageData | NDArray):
             self.get_segment_labelmap(segment_id, as_numpy_array=True)[:] = label_map
         self.segmentation_modified()
 
+    def set_segment_visibility(self, segment_id, visibility: bool):
+        if segment_id not in self.get_segment_ids():
+            return
+
+        if not self._segmentation_node:
+            return
+
+        display_node = self._segmentation_node.GetDisplayNode()
+        display_node.SetSegmentVisibility(segment_id, visibility)
+
+    def get_segment_visibility(self, segment_id):
+        if segment_id not in self.get_segment_ids():
+            return None
+
+        if not self._segmentation_node:
```

Comment:
Check is redundant. If segmentation node is unset, segment_id will not be in the segment ids (return value = [])